.github/workflows/on_build_do_test.yml

```diff
@@ -338,6 +346,8 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           min-coverage-overall: 70
           min-coverage-changed-files: 70
+          pr-number: ${{ env.PR_NUMBER }}
+          debug-mode: true
```

Comment:
Do we need this enabled now that you have finished testing?

Reply:
I wanted to have this enabled just in case it will have some other glitches. It prints a few logs so shouldn't spam us.

Reply:
Ok np